src/common/github-flavored-markdown.js

```diff
@@ -66,6 +76,28 @@ class GithubFlavoredMarkdown {
     } });
     return [...linkSet];
   }
+
+  redactCdnResources(markdown, cb) {
+    if (!markdown) {
+      return markdown;
+    }
+
+    const redact = url => {
+      if (url.startsWith('cdn://')) {
+        const pathOnly = url.slice(6);
```

Comment:
Maybe we can extract
`const cdnUrlBasePath = 'cdn://'`
and use it in
`const pathOnly = url.slice(cdnUrlBasePath.length)`
so that this 6 is not a magic number and to reduce the risk of a nasty bug if the base url ever changes again 😬

Reply:
Greate, idea, done, thanks!